js/ui/new-metric-popup.js

```diff
@@ -0,0 +1,79 @@
+import { PopupHeader } from './popup-header.js'
+import { veil } from './veil.js'
+import { Store } from '../store.js'
+import { markerSymbols, Metric } from '../metric.js'
+
+export const NewMetricPopup = {
+  components: {
+    PopupHeader
+  },
+  model: {
+    prop: 'popupStatus',
+    event: 'toggle'
+  },
+  props: {
+    popupStatus: {
+      type: Boolean,
+      required: true
+    }
+  },
+  data: function () {
+    return {
+      popupTitle: 'Metriken hinzufügen',
+      value: null,
+      options: []
+    }
+  },
+  mounted () {
+    const popupEle = document.querySelector('.new_metric_popup_div')
+    if (popupEle) {
+      const disablePopupFunc = () => {
+        this.$emit('toggle', false)
+        window.MetricQWebView.instances[0].reload()
+      }
+      veil.create(disablePopupFunc)
+      veil.attachPopup(popupEle)
+    }
+  },
+  methods: {
+    addMetrics: function (evt) {
+      if (this.value != null) {
+        this.value.forEach(function (item, index, array) {
+          Store.setMetric(item, new Metric(window.MetricQWebView.instances[0], item, undefined, markerSymbols[(index * 4) % markerSymbols.length], []))
```

Comment:
I don't like the selection of markerSymbols at this point. I'd rather leave it `undefined` and have `Metric.constructor` pick one for us at "random" / based on the metric name, just like we do it with the color.

The same applies to the other point where we call `new Metric` - it's even worse there with no modulo wrap around.